src/puppetlabs/trapperkeeper/bootstrap.clj

```diff
@@ -8,16 +8,20 @@
             [puppetlabs.trapperkeeper.services :as services]
             [puppetlabs.trapperkeeper.common :as common]
             [schema.core :as schema]
-            [me.raynes.fs :as fs]))
+            [me.raynes.fs :as fs]
+            [slingshot.slingshot :refer [try+ throw+]]))
 
 ;; Schemas
-(def BootstrapLine {:namespace schema/Str :service-name schema/Str})
+(def ParsedBootstrapEntry {:namespace schema/Str :service-name schema/Str})
 (def BootstrapFiles [(schema/protocol io/IOFactory)])
+(def AnnotatedBootstrapEntry {:entry schema/Str
+                    :bootstrap-file schema/Str
+                    :line-number schema/Int})
 
 ;; Constants
 (def bootstrap-config-file-name "bootstrap.cfg")
 
-(schema/defn ^:private parse-bootstrap-line! :- BootstrapLine
```

Comment:
If it's easier you can do that re-organization in a follow-up PR.

Personally I'm fine with keeping the `^:private` metadata tags until they cause a problem for a given fn, but I'm also fine with removing them if we add the pragma stuff.